meson.build

```diff
@@ -260,7 +262,11 @@ if get_option('bash-completions')
 		'completions/bash/swaybar',
 		'completions/bash/swaymsg',
 	)
-	bash_install_dir = datadir + '/bash-completion/completions'
+	if bash_comp.found()
+		bash_install_dir = bash_comp.get_pkgconfig_variable('completionsdir')
+	else
+		bash_install_dir = datadir + '/bash-completion/completions'
```

Comment:
Use join_paths instead of +

Reply:
I suppose I could do that, but the current implementation is just shifting over existing code into a conditional. I wasn't sure what the original intent of using string concat rather than join_paths() was. :)

Reply:
I've gone ahead and fixed this as well as a few other instances in a preliminary patch.